pkg/alertmanager/alertmanager.go

```diff
@@ -529,7 +569,11 @@ func buildIntegrationsMap(nc []*definition.PostableApiReceiver, tmpl *template.T
 	return integrationsMap, nil
 }
 
-func buildGrafanaReceiverIntegrations(rcv *definition.PostableApiReceiver, tmpl *template.Template, logger log.Logger) ([]*nfstatus.Integration, error) {
+func (am *Alertmanager) BuildGrafanaReceiverIntegrations(rcv *alertingNotify.APIReceiver, tmpl *template.Template) ([]*nfstatus.Integration, error) {
+	return buildGrafanaReceiverIntegrations(rcv, tmpl, am.logger)
+}
```

Comment:
I'd make this method private, we don't need to call it outside of this struct.
```suggestion
func (am *Alertmanager) buildGrafanaReceiverIntegrations(rcv *alertingNotify.APIReceiver, tmpl *template.Template) ([]*nfstatus.Integration, error) {
	return buildGrafanaReceiverIntegrations(rcv, tmpl, am.logger)
}
```